format-parquet/src/test/java/io/cdap/plugin/format/parquet/input/ParquetInputFormatProviderTest.java

```diff
@@ -74,4 +74,38 @@ public void testSchemaDetection() throws IOException {
     Schema schema = schemaDetector.detectSchema(parquetFile.getAbsolutePath(), formatContext, Collections.emptyMap());
     Assert.assertEquals(expectedSchema, schema);
   }
+
+  @Test
+  public void testPathFieldSchemaDetection() throws IOException {
+    Configuration hConf = new Configuration();
+    File parquetFile = new File(TMP_FOLDER.newFolder(), "test.parquet");
+    Path parquetPath = new Path(parquetFile.toURI());
+
+    Schema expectedSchema = Schema.recordOf("x",
+            Schema.Field.of("id", Schema.of(Schema.Type.INT)),
+            Schema.Field.of("name", Schema.of(Schema.Type.STRING)),
+            Schema.Field.of("score", Schema.of(Schema.Type.DOUBLE)),
+            Schema.Field.of("pathField", Schema.of(Schema.Type.STRING)));
+    org.apache.avro.Schema avroSchema = new org.apache.avro.Schema.Parser().parse(expectedSchema.toString());
+    GenericRecord record = new GenericRecordBuilder(avroSchema)
+            .set("id", 0)
+            .set("name", "alice")
+            .set("score", 1.0d)
+            .set("pathField", "value")
```

Comment:
similar comment to the one from the avro test

Reply:
Changed such that exception arises if  pathfield value which is already existing in schema is given